Example/IGListKitExamples/SectionControllers/SpinnerSectionController.swift

```diff
@@ -0,0 +1,37 @@
+/**
+ Copyright (c) 2016-present, Facebook, Inc. All rights reserved.
+ 
+ The examples provided by Facebook are for non-commercial testing and evaluation
+ purposes only. Facebook reserves all rights not expressly granted.
+ 
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import UIKit
+import IGListKit
+
+class SpinnerSectionController: IGListSectionController, IGListSectionType {
+    
+    func numberOfItems() -> Int {
+        return 1
+    }
+    
+    func sizeForItem(at index: Int) -> CGSize {
+        return CGSize(width: collectionContext!.containerSize.width, height: 100)
+    }
+    
+    func cellForItem(at index: Int) -> UICollectionViewCell {
+        let cell = collectionContext!.dequeueReusableCell(of: SpinnerCell.self, for: self, at: index) as! SpinnerCell
+        cell.activityIndicator.startAnimating()
```

Comment:
Could we just update the function using `IGListSingleSectionController` to call `startAnimating()` in the `configureBlock(...)` instead of making a new section controller? 

I just want to make sure we have an example using the single section controller somewhere.